context/defaultSkuConfig.js

```diff
@@ -1,5 +1,6 @@
 const supportedBrowsers = require('browserslist-config-seek');
 const path = require('path');
+const isTypeScript = require('../lib/isTypeScript');
```

Comment:
`isTypeScript` is actually a function. However, it probably should be a value because otherwise, we may call it multiple times and it's not zero cost.